cypress/integration/Select.spec.ts

```diff
@@ -182,381 +577,3 @@ describe('Select', () => {
-              it('should set assistive focus to the "San Francisco (United States)" option', () => {
-                cy.findByText('San Francisco (United States)').should(
-                  'have.attr',
-                  'aria-selected',
-                  'true'
-                );
-              });
-            });
-          });
-
-          context('when "san " is typed', () => {
-            beforeEach(() => {
-              cy.findByRole('combobox').realType('san ');
-            });
-
-            context('the select input', () => {
-              it('should set assistive focus to the "San Francisco (United States)" option', () => {
-                cy.findByText('San Francisco (United States)').should(
-                  'have.attr',
-                  'aria-selected',
-                  'true'
-                );
-              });
-            });
-          });
-
-          context('when "san m" is typed', () => {
-            beforeEach(() => {
-              cy.findByRole('combobox').focus().realType('san m');
-              cy.wait(150);
-            });
-
-            context('the select input', () => {
-              it('should set assistive focus to the "San Mateo (United States)" option', () => {
-                cy.findByText('San Mateo (United States)').should(
-                  'have.attr',
-                  'aria-selected',
-                  'true'
-                );
-              });
-            });
-          });
-        }
-      );
-    });
-
-    context(
-      'when the menu is opened and the selected option is initially out of view, the menu should scroll the selected option into view and center it if possible',
-      () => {
-        context('when "Dallas (United States)" is selected and the menu is opened', () => {
-          beforeEach(() => {
-            cy.findByRole('combobox').focus().type('d').click();
-          });
-
-          context('the menu', () => {
-            it('should scroll so that the "Dallas (United States)" option is centered in view', () => {
-              cy.findByText('Dallas (United States)').should('have.attr', 'aria-selected', 'true');
-            });
-          });
-        });
-      }
-    );
-  });
-});
-
-context(`given the "FetchingDynamicItems" story is rendered`, () => {
-  before(() => {
-    h.stories.visit();
-  });
-  beforeEach(() => {
-    h.stories.load('Components/Inputs/Select', 'FetchingDynamicItems');
-  });
-
-  context('when Get Items is clicked', () => {
-    beforeEach(() => {
-      cy.findByRole('button', {name: 'Get Items'}).click();
-    });
-    it('should change the value of the select to 456 (the id) after 1.5 seconds', () => {
-      cy.wait(1500);
-      cy.findByRole('combobox').should('have.value', 'John Wick');
-      cy.findByTestId('selected-value').should('have.text', 'Selected value: John Wick');
-      cy.findByTestId('selected-id').should('have.text', 'Selected Id: 456');
-    });
-  });
-});
```

Comment:
Are these existing tests?

Reply:
yep, just moved them around to order them the way Raisa suggested